### PR TITLE
feat(css/prefixer): font-face format

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

Port https://preset-env.cssdb.org/features/#font-format-keywords

**BREAKING CHANGE:**

No

**Related issue (if exists):**

No

### CODE REVIEW

File: crates/swc_css_prefixer/tests/fixture/font-face-format/output.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+@font-face{
+  src: format("woff"), format("truetype"), format("opentype"), format("woff2"), format("embedded-opentype"), format("collection"), format("svg");
+  src: format(woff), format(truetype), format(opentype), format(woff2), format(embedded-opentype), format(collection), format(svg);
+}
+@Font-face{
+  src: FORMAT("WOFF");
+  src: FORMAT(WOFF);
+}
+@font-face{
+  src: format(url("woff"));
```
Comment (alexander-akait) [inline crates/swc_css_prefixer/tests/fixture/font-face-format/output.css:18]: Weird bug, WIP

Comment (alexander-akait) [reply crates/swc_css_prefixer/tests/fixture/font-face-format/output.css]: Found a bug :smile: fixed

**Review by kdy1 — APPROVED** [review]
swc-bump:
 - swc_css_prefixer

**Review by kdy1 — CHANGES_REQUESTED** [review]
Merged failed

**Comment by alexander-akait** [comment]
@kdy1 Rebased and fixed

File: crates/swc_css_prefixer/src/prefixer.rs [hunk 2]
```diff
@@ -591,6 +591,53 @@ where
...
+        if !n.name.value.eq_ignore_ascii_case(&js_word!("format")) {
+            return;
+        }
+
+        if n.value.len() != 1 {
+            return;
+        }
+
+        if let Some(ComponentValue::Ident(box ident)) = n.value.get(0) {
+            let new_value: JsWord = ident.value.clone().to_ascii_lowercase();
```
Comment (kdy1) [inline crates/swc_css_prefixer/src/prefixer.rs:609]: I think this `clone()` is redundant

**Review by kdy1 — APPROVED** [review]
Thank you!

**Pull request merged**